hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/BucketLayout.java

```diff
@@ -70,6 +70,10 @@ public boolean isLegacy() {
     return this.equals(LEGACY);
   }
 
+  public boolean isObjectStore() {
+    return this.equals(OBJECT_STORE);
```

Comment:
A legacy bucket with "ozone.om.enable.filesystem.paths" set to false is also an object store bucket.  so test for that here as well.